src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx

```diff
@@ -238,21 +243,28 @@ function NodeComponent({
                                 </div>
 
                                 <div>
-                                    <Tippy
-                                        className="default-tt"
-                                        arrow={false}
-                                        placement="bottom"
-                                        content={copied ? 'Copied!' : 'Copy to clipboard.'}
-                                        trigger="mouseenter click"
-                                    >
+                                {node?.name ? copied[node?.name] ? (
```

Comment:
why this conditional chaining?

Reply:
I mapped the isCopied state with the node name to a boolean. So the copy clipboard should only appear if there's a name to copy. That way it won't map an empty string (i.e. the node name for this case) to boolean value.
``` the basic idea was to available copy feature only if there's a name to copy```

Reply:
updated 👍